raiden/tests/unit/transfer/mediated_transfer/test_targetstate.py

```diff
@@ -638,7 +639,7 @@ def test_target_lock_is_expired_if_secret_is_not_registered_onchain():
         block_number=block_number,
     )
 
-    expired_block_number = from_transfer.lock.expiration + DEFAULT_NUMBER_OF_BLOCK_CONFIRMATIONS
+    expired_block_number = channel.get_sender_expiration_threshold(from_transfer.lock)
```

Comment:
I did not read the test, but the expression was for the receiver, not the sender